server/views/pages/prisons/viewSingleSessionTemplate.njk

```diff
@@ -68,6 +69,16 @@
 
       {{ govukSummaryList({
         rows: [
+          {
+          key: {
+            text: "Status"
+          },
+          value: {
+            text: changeStatusActionText,
```

Comment:
Could put this instead of `text` to get GOVUK style tag (like we have for active/inactive prison):
```
html: govukTag({
              text: "Active" if sessionTemplate.active == true else "Inactive",
              classes: "govuk-tag--blue" if sessionTemplate.active else "govuk-tag--grey"
            }),
```

Would also need to add import at the top:
```
{%- from "govuk/components/tag/macro.njk" import govukTag -%}
```